sdks/go/pkg/beam/transforms/sql/sql_test.go

```diff
@@ -0,0 +1,105 @@
+package sql
+
+import (
+	"github.com/apache/beam/sdks/v2/go/pkg/beam"
+	"github.com/apache/beam/sdks/v2/go/pkg/beam/transforms/sql/sqlx"
+	"reflect"
+	"runtime"
+	"testing"
+)
+
+func TestOptions_Add(t *testing.T) {
+	o := options{}
```

Comment:
Defining this outside of the loop means additional test cases are not going to be hermetic. They will depend on each other. Test case independence is preferred. in this case, it means adding it to the top of the loop.